instance/tests/models/test_openedx_monitoring_mixins.py

```diff
@@ -336,6 +342,9 @@ def test_email_address_deletion_on_disabling_monitoring(self, mock_newrelic):
         NewRelicEmailNotificationChannel.objects.get(email='admin@opencraft.com')
 
     def test_deleting_shared_notification_email_address(self):
+        """
+        Test that a shared email notification channel can't be deleted.
```

Comment:
@clemente Shared notification channels can't be deleted because they are shared between many instances.
For example: `ops@opencraft.com` will be a shared notification channel that will be used for NewRelic alerts from all instances. `urgent@opencraft.com` is also another alert that is shared by all maintenance instances.  We can't delete them because if we do, all other instances that depend on that notification channel will also be deleted.

This was made this way because how NewRelic alerts works, and we can't add two of the same email address to NewRelic Alerts.

Reply:
Thanks, I added this comment to the test, for more context.